src/main/java/objective/SuspiciousFile.java

```diff
@@ -0,0 +1,2 @@
+System.out.println('credentials');
+System.out.println('password=1234');
```

Comment:
Suspeita de credencial encontrada, tipo: Password

Se for um falso positivo, adicione o seguinte na mensagem do commit (amend) para que seja ignorada:
```
FalsoPositivo: 4df28cec
```